### PR TITLE
Balance

### PR DESCRIPTION
**Pull request by l34Um1**

### CODE REVIEW

**Comment by GymLeaderGiovanni** [comment]
No change to -10 second io relocate cool down?

**Comment by l34Um1** [comment]
It's changed, check again.

**Comment by l34Um1** [comment]
I don't know why this is showing an error.

File: game/scripts/npc/abilities/swiper/boss_swiper_frontswipe.txt [hunk 1]
```diff
@@ -22,12 +22,12 @@
...
         //-------------------------------------------------------------------------------------------------------------
         "AbilityCastRange"              "300"
-        "AbilityCastPoint"              "2.2"
+        "AbilityCastPoint"              "2"
         // "Radius"                        "200"
 
         // Time
         //-------------------------------------------------------------------------------------------------------------
-        "AbilityCooldown"               "2"
+        "AbilityCooldown"               0.5"
```
Comment (Trildar) [inline game/scripts/npc/abilities/swiper/boss_swiper_frontswipe.txt]: Missing open quote on the value.

Comment (l34Um1) [reply game/scripts/npc/abilities/swiper/boss_swiper_frontswipe.txt]: Thanks

**Pull request merged**